ui/src/markdown.css

```diff
@@ -0,0 +1,720 @@
+/*
+  ! tailwindcss v3.3.5 | MIT License | https://tailwindcss.com
+*/
+/*
+1. Prevent padding and border from affecting element width. (https://github.com/mozdevs/cssremedy/issues/4)
+2. Allow adding a border to an element by just adding a border-width. (https://github.com/tailwindcss/tailwindcss/pull/116)
+*/
+*,
+::before,
```

Comment:
Let's scope these styles to the markdown component only - do not leak them into global namespace.